lib/modules/manager/terragrunt/modules.ts

```diff
@@ -87,7 +87,10 @@ export function analyseTerragruntModule(
       }
     } else {
       // The packageName should only contain the path to the repository
-      dep.packageName = pathname.replace(/^\//, '').split('//')[0];
+      dep.packageName = pathname
```

Comment:
please validate in test

Reply:
Did [here]( https://github.com/renovatebot/renovate/pull/28726/files#diff-a1f325dbaaba072ae6fb920e3c3edfb9d31621e9aeb4f6fed022536ecc8e415cR185) already, or do you want to have a separate test case?

Reply:
update that test and validate passed args to the get digest function

Reply:
Sorry, I'm not quite sure if I understood correctly. Which test do you want me to update?

I've added the `.git` to the test-case using a ssh URL as source.  The expected `packageName` already has been `hasicorp/example` (see [here](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/terragrunt/extract.spec.ts#L238)). So this case proves, that the `.git` part is ignored.